### PR TITLE
Run cargo clippy --fix

### PR DESCRIPTION
**Pull request by sisuresh**

### CODE REVIEW

File: soroban-env-host/src/host.rs [hunk 1]
```diff
@@ -952,11 +956,11 @@ impl Host {
             }
             HostFunction::CreateContract(args) => self
                 .with_frame(Frame::HostFunction(hf_type), || {
-                    self.create_contract(args).map(|obj| <RawVal>::from(obj))
+                    self.create_contract(args).map(<RawVal>::from)
                 }),
             HostFunction::InstallContractCode(args) => self
                 .with_frame(Frame::HostFunction(hf_type), || {
-                    self.install_contract(args).map(|obj| <RawVal>::from(obj))
+                    self.install_contract(args).map(<RawVal>::from)
```
Comment (leighmcculloch) [inline soroban-env-host/src/host.rs:963]: Why are the `<` `>` required around `RawVal` here? I think they can be omitted.

**Review by leighmcculloch — APPROVED** [review]
👏🏻

File: soroban-env-host/src/host.rs (hunk 1 above)
Comment (sisuresh) [reply soroban-env-host/src/host.rs]: Yeah you're right. Removed.

**Comment by sisuresh** [comment]
Closing for now because we still have https://github.com/stellar/rs-soroban-env/pull/600/ pending, and don't want to cause more merge conflicts in that PR. I'll reopen this when we don't have any large pending changes.

**Pull request closed without merging**